### PR TITLE
Bugfixes

### LINKED ISSUE #6: Check for exported directory
We should make a check for existence of ```exported```directory.
If it doesn't exist - it should be created.

image

### LINKED ISSUE #7: Update task-status JSON
Now we have such fields in task-status json:
`Time | User | Datasource | Measurement | Exported Rows | Progress | Download CSV | Status | Delete`
Measurement is only influxdb-related field and should be removed
Also, datasource field is always empty now, should change the way it is filled

### LINKED ISSUE #8: Change exported filenames
We have `measurement` as part of exported filename now
It's influxdb-only field, should use something not datasource-specific

### PR DESCRIPTION
**Pull request by rozetko**
- Use datasource name instead of measurement (closes #7)
- Change exported filenames (closes #8)
- Rename export-manager-backend -> grafana-data-exporter
- Check for exported directory (fixes #6)

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -2,7 +2,10 @@ import * as path from 'path';
 import * as fs from 'fs';
 
 export const EXPORTED_PATH = path.join(__dirname, '../exported');
-
+if(!fs.existsSync(EXPORTED_PATH)) {
+  console.log(`${EXPORTED_PATH} don't exist, creating`);
+  fs.mkdirSync(EXPORTED_PATH);
```
Comment (amper43) [inline src/config.ts]: what about exceptions becaufe of file system rights?

File: src/target.ts [hunk 2]
```diff
@@ -105,9 +107,7 @@ export class Target {
   }
 
   private getFilename(extension) {
-    // TODO: use something unique instead of measurement in filename
-    // as measurement field exists only in influxDB metric
-    return `${this.metric.targets[0].measurement}.${this.from}-${this.to}.${extension}`;
+    return `${this.datasourceName}.${this.from}-${this.to}.${extension}`;
```
Comment (amper43) [inline src/target.ts]: if we create two exports for different metrics but for same datasource, will we rewrite old export file for same time range?

Comment (rozetko) [reply src/target.ts]: Unfortunately
Didn't come up with the metric-unique field yet
I would create issue about that, thx

File: src/config.ts (hunk 1 above)
Comment (rozetko) [reply src/config.ts]: Actually, we can't manage this exception anyhow
So, maybe it's better to just let it fall?

Comment (amper43) [reply src/config.ts]: ok

File: src/target.ts (hunk 2 above)
Comment (amper43) [reply src/target.ts]: may be it would be better to use datasourceName.current epoch time.extension ?

Comment (rozetko) [reply src/target.ts]: good idea 👍

**Pull request merged**